react/components/AdminMessages.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useState } from 'react'
+import { InjectedIntl, injectIntl } from 'react-intl'
+
+import { Layout, PageHeader } from 'vtex.styleguide'
+
+import { AddProductTranslationsMutationFn } from '../mutations/AddProductTranslations'
+import { MessagesOfProvider, StepCounterControl } from '../typings/typings'
+import ProductMessagesExport from './ProductMessagesExport'
+import ProductMessagesImport from './ProductMessagesImport'
+
+const TOTAL_STEPS = 3
+
+interface Props {
+  intl: InjectedIntl
+  addProductTranslations: AddProductTranslationsMutationFn
+  email: string
+}
+
+const AdminMessages: FC<Props> = ({ email, intl }) => {
+  const [step, setStep] = useState(1)
+
+  const translationDataHooks = {
+    language: useState(''),
+    messages: useState<MessagesOfProvider[] | null>(null),
+  }
+
+  const next = () => setStep(step + 1)
+  const back = () => setStep(step - 1)
+
+  const stepCounterControl = ({
+    back,
+    current: step.toString(),
+    next,
+    total: TOTAL_STEPS.toString(),
+  } as unknown) as StepCounterControl
+
+  return (
+    <div className="min-vh-100 bg-muted-5">
+      <Layout
+        pageHeader={
+          <PageHeader
+            title={intl.formatMessage({ id: 'admin/messages.headerTitle' })}
+          />
+        }
+      >
+        {step !== 1 ? null : (
```

Comment:
This seems kinda weird, what I usually see is `step === 1 ? <Component> : null` instead of the negative `step !== 1 ? null : <Component`. WDYT?

Reply:
For me is kind of hard to see which ternary the `null` relates too, mainly when there is a big component over it.